server/templates/staff/grading/code.html

```diff
@@ -23,7 +24,7 @@
           <i class="fa fa-list"></i> Assignments</a>
         </li>
         <li class="active"><a href="{{ url_for('.assignment', cid=backup.assignment.course.id, aid=backup.assignment.id) }}"><i class="fa fa-book"></i> {{ backup.assignment.display_name }}</a></li>
-        <li class="active"><a href="{{ url_for('.grading', bid=backup.id) }}"><i class="fa fa-book"></i> Grade {{ utils.encode_id(backup.id) }}</a></li>
+        <li class="active"><a href="{{ url_for('.grading', bid=backup.id) }}"><i class="fa fa-book"></i> Grade {{ globals.backup_style(utils.encode_id(backup.id)) }}</a></li>
```

Comment:
It's a little tricky because I'm using a `caller` block [in the macro](https://github.com/Cal-CS-61A-Staff/ok/pull/1004/files#diff-e77070762b75355859a1e56d482601d1R49). But the `caller` block [used here](https://github.com/Cal-CS-61A-Staff/ok/pull/1004/files#diff-f36f46a206b18c1a9ab5d05948aaf572R210) needs to place the green star after the link whereas, in this bit of code, we need to prepend the book icon and "Grade".

Any suggestions?